.github/workflows/ci.yaml

```diff
@@ -0,0 +1,20 @@
+---
+on: [push, pull_request]
+name: CI
+jobs:
+  # RELEASE: runs after a success test only runs on push "v*" tag
+  release_binaries:
+    name: Release Binaries
+    if: startsWith(github.ref, 'refs/tags/v')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: goreleaser
+        if: success()
+        uses: docker://goreleaser/goreleaser:latest
```

Comment:
Any benefits from using the docker instead of the official [github action goreleaser](https://github.com/marketplace/actions/goreleaser-action)?

Reply:
@dtdao 
I don't think there is a significant benefit between Docker and a GitHub Actions repository.
I've tried GitHub Actions already, and it worked.